services/postgres/alembic/versions/2020-10-21T141127_add_artifact_api.py

```diff
@@ -0,0 +1,53 @@
+"""
+Add artifact api
+
+Revision ID: 3c87ad7e0b71
+Revises: 24f10aeee83e
+Create Date: 2020-10-21 14:11:27.409161
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+
+
+# revision identifiers, used by Alembic.
+revision = "3c87ad7e0b71"
+down_revision = "24f10aeee83e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
```

Comment:
We should add an updated column + trigger to this table as well

Reply:
When I spoke to @jlowin about that column he advised against adding it

Reply:
I think I may have inadvertently created confusion here -- we do need this column/trigger on every table; for the most highly-trafficked tables (which this will certainly not be), that can create heavy update contention which has to be dealt with separately.

Reply:
Okay awesome, thanks for clarifying 👍  will add this column/trigger